crates/cargo-contract/src/cmd/build/mod.rs

```diff
@@ -100,6 +108,9 @@ pub struct BuildCommand {
     /// Then no debug functionality is compiled into the contract.
     #[clap(long = "--release")]
     build_release: bool,
+    /// Build all contract packages in the workspace
+    #[clap(long = "--all")]
```

Comment:
Cargo has deprecated `--all` in favour of `--workspace`, we should attempt to be consistent with the `cargo` flags as far as possible

![image](https://user-images.githubusercontent.com/75586/192001582-125a50a5-6d51-495e-9e3f-ecb140a19e0e.png)